### PR TITLE
Feat/5818 remove borders and update colors

### PR DESCRIPTION
**Pull request by gbonaspetti**
#5818

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
  - This PR just change some colors and borders, no tests needed for that.
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by Haskaris** [comment]
I just went for a little try before any review/test
Did we have considered removing some colors?

For example:
image
To
image

The white block made sense when we had borders since buttons were white too. But since they aren't white anymore, maybe we can remove the "block" background to reduce the interface weight

**Comment by gbonaspetti** [comment]
@Haskaris Thanks for all the feedbacks, they will be taken into account for the improvements of this PR, the idea is to make a first version that even not being perfect, can be merged today so we can test it during the week and make the improvements pointed out.

File: frontend/src/component/Activity/ActivityList.styl [hunk 1]
```diff
@@ -20,7 +20,7 @@ previewHeight = 400px
 
     &__overflowOverlay
       background-image:
-        linear-gradient(to top, rgba(0, 0, 0, 0.25), transparent),
+        linear-gradient(to top, rgba(0, 0, 0, 0.1), transparent),
```
Comment (Haskaris) [inline frontend/src/component/Activity/ActivityList.styl:23]: `linear-gradient(to top, rgba(0, 0, 0, 0.1), transparent)` should be `linear-gradient(to top, rgba(1, 1, 1, 1), transparent)`
So the gradient will fade to the white

Comment (gbonaspetti) [reply frontend/src/component/Activity/ActivityList.styl:23]: `rgba(1, 1, 1, 1)` it's black, to fade to white it's `rgba(255, 255, 255, 1)`. I can update this, the initial change was just "make it a little closer to the gray of the card".

**Comment by Millefeuille42** [comment]
About the emojis reactions, to make the reactions User did to stand-out I'd suggest using a distinct color, and not playing with shades. For example, a shade of main color or success color could be used, using the alpha layer to give a soft but distinguishable color

**Comment by gbonaspetti** [comment]
Yes, we use the primary color. After it was a lighten color of the primary, but after the comment I changed to use an opacity of it

**Comment by Millefeuille42** [comment]
My bad I did not see the changes before commenting, this is indeed way better. Unfortunately this is still improvable, but the current color palette is not permitting better so I think it's OK

**Comment by Millefeuille42** [comment]
Tested on branch

**Review by Millefeuille42 — APPROVED** [review]

**Pull request merged**